.github/workflows/latest_deps.yml

```diff
@@ -26,12 +26,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
```

Comment:
For latest deps, do we want stable or nightly? Stable is probably the closest to what we do for Python packages.

Reply:
Yeah, we could also do beta I guess? I don't think it matters much

Reply:
Sure. (Just thinking out loud here.)